Update uswds-theme.scss references (#1866)
uswds/uswds-site: _utilities/layout-grid.md

{:.usa-content-list}
- **Containers:** `grid-container` centers the container and gives it a maximum width of `desktop` (1024px). If you would like the grid to span the full width of the page, do not use `grid-container`.

>   - `grid-container` can also accept any breakpoint width, like `grid-container-tablet-lg` or `grid-container-widescreen`. Set the default max width with `$theme-grid-container-max-width` in `uswds-theme.scss`.

Reviewer: Forgot leading underscore in `_uswds-theme.scss`.

>   - `grid-container` can also accept any breakpoint width, like `grid-container-tablet-lg` or `grid-container-widescreen`. Set the default max width with `$theme-grid-container-max-width` in your [USWDS settings configuration]({{ site.baseurl }}/documentation/settings) using [unit tokens]({{ site.baseurl }}/design-tokens/spacing-units/).

Reviewer 2: These values might be outdated?
```suggestion
  - `grid-container` can also accept any breakpoint width, like `tablet-lg` or `widescreen`. Set the default max width with `$theme-grid-container-max-width` in your [USWDS settings configuration]({{ site.baseurl }}/documentation/settings) using [unit tokens]({{ site.baseurl }}/design-tokens/spacing-units/).
```

>   - By default, `grid-container`s have padding-x of [2 units]({{ site.baseurl }}/design-tokens/spacing-units/){:.token}, with a padding-x of [4 units]({{ site.baseurl }}/design-tokens/spacing-units/){:.token} at `desktop` and wider. Control these values with the values of `$theme-site-margins-mobile-width`, `$theme-site-margins-width` and `$theme-site-margins-breakpoint` in your [USWDS settings configuration]({{ site.baseurl }}/documentation/settings) using [unit tokens]({{ site.baseurl }}/design-tokens/spacing-units/).

Reviewer 2: Optional: some additions to clarify meaning
```suggestion
  - By default, `grid-container` has a `padding-x` of [2 units]({{ site.baseurl }}/design-tokens/spacing-units/){:.token} at narrow widths, and a padding-x of [4 units]({{ site.baseurl }}/design-tokens/spacing-units/){:.token} at `desktop` and wider. Control these values by customizing `$theme-site-margins-mobile-width`, `$theme-site-margins-width` and `$theme-site-margins-breakpoint` in your [USWDS settings configuration]({{ site.baseurl }}/documentation/settings) using [unit tokens]({{ site.baseurl }}/design-tokens/spacing-units/).
```

When generating your CSS from USWDS source files, you have the option of customizing many system defaults by modifying project theme variables. USWDS also provides grid mixins for adding grid functionality to custom semantic component CSS.

### Variables

> Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under Spacing settings and Utilities settings

Reviewer 3: Can you link to spacing settings and utilities settings (even if they're in-page)?

Author: Sure! They are _directly_ beneath this paragraph but as long as you're cool with that!

Author: Resolved in https://github.com/uswds/uswds-site/pull/1866/commits/174fb5284a163aab653d6b1a843d393ae8a56be8

> Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under [Spacing settings](#spacing-settings) and [Utilities settings](#utilities-settings)

Reviewer: Missing period at end of sentence.

![image](https://user-images.githubusercontent.com/3385219/200364918-2a4069e2-ebdc-4cda-b65c-e3acd358eb2f.png)
